tests/include.am

```diff
@@ -8,20 +8,20 @@ noinst_PROGRAMS += tests/unit.test tests/api.test \
                    tests/testsuite.test
 
 tests_unit_test_SOURCES      = tests/unit.c
-tests_unit_test_CFLAGS       = -DNO_MAIN_DRIVER $(AM_CFLAGS)
-tests_unit_test_LDADD        = src/libwolfssh.la $(LIB_STATIC_ADD)
+tests_unit_test_CPPFLAGS     = -DNO_MAIN_DRIVER
+tests_unit_test_LDADD        = src/libwolfssh.la
```

Comment:
why remove the additional static librarys here? will this hinder cases where testing with memory tools that require linking against a third party library?

Reply:
I don't believe LIB_STATIC_ADD is common or recommended anywhere except in wolfSSL's Makefile.am. I understand the common practice is to add "LDFLAGS=-llibraryname" to the configure command to add any required libraries.